astropy/units/format/console.py

```diff
@@ -92,6 +82,14 @@ def _format_fraction(
     def to_string(
         cls, unit: UnitBase, fraction: bool | Literal["inline", "multiline"] = False
     ) -> str:
-        # Change default of fraction to False, i.e., we typeset
```

Comment:
`Console.to_string()` cannot stay as it is anyways because at the very least it needs to ensure that when an incorrect `fraction` value is passed in then the error message correctly lists which values are supported.